### PR TITLE
Ivan/i18n

### PR DESCRIPTION
**Pull request by ivan-dalmet**
Add [Next Translate](https://github.com/vinissimus/next-translate) setup

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (yoannfleurydev) [inline .editorconfig:12]: Funny thing, you have set the `insert_final_newline` to `true` and this file is missing the `insert_final_newline` 🤭

Comment (ivan-dalmet) [reply .editorconfig:12]: I copy/paste it from @DecampsRenan PR 😅

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (yoannfleurydev) [inline .vscode/settings.json:1]: I'm not a huge fan of putting the `settings.json` in the repository as I often override it myself in every project with specific settings. Are these configurations really useful?

**Review by yoannfleurydev — APPROVED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (ivan-dalmet) [reply .vscode/settings.json]: I've removed the settings.json and added the extensions.json ;)

**Pull request merged**